### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by oriyannadav**
The footer is all ready and is media responsive. 

If anyone has any input on how to make the footer better looking, please let me know. 

I also had to make some minor changes to the navbar to be responsive as well.

### CODE REVIEW

File: src/components/Footer/Footer.css
Comment (rosiejeays) [inline src/components/Footer/Footer.css:1]: Are any of these being reused elsewhere?
Is there a plan to refactor the CSS once the pages and components are all in development?

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                <Link 
+                className="icon"
+                to="https://www.linkedin.com/company/shecodesaustralia/" 
+                id="linkedin"
+                >
+                    <img src={LinkedInIcon} alt="" />
+                </Link>
+                <Link 
+                className="icon"
+                to="https://join.slack.com/t/shecodesaus/shared_invite/enQtNzQ3ODk1OTQzODc3LTQ2ZWFmNzM0N[…]JkMWMyYWVjMDU3ZThhODJmMTk4YmRjZGMzZDRhZDBhODQwNzdlYmJhNDg1YjE"
```
Comment (rosiejeays) [inline src/components/Footer/Footer.jsx]: This link no longer works. Might be better to update with the one used on the She Codes website:
https://shecodesaus.slack.com/join/shared_invite/enQtNzQ3ODk1OTQzODc3LTQ2ZWFmNzM0NjE1MjA0NTI3YjJkMWMyYWVjMDU3ZThhODJmMTk4YmRjZGMzZDRhZDBhODQwNzdlYmJhNDg1YjE#/shared-invite/email

File: src/components/NavBar/NavBar.css
Comment (rosiejeays) [inline src/components/NavBar/NavBar.css:1]: Are any of these being reused elsewhere?
Is there a plan to refactor the CSS once the pages and components are all in development?

File: src/pages/HomePage/HomePage.css
Comment (rosiejeays) [inline src/pages/HomePage/HomePage.css:1]: Are any of these being reused elsewhere?
Is there a plan to refactor the CSS once the pages and components are all in development?

File: src/pages/HomePage/HomePage.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                    ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco 
+                    laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in 
+                    voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat 
+                    non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                    </p>
+                    <button className="button">More Info</button>
+                </div>
+                <div className="next-event">
+                    <h1>Next Event</h1>
+                    <div className="paragraph">
```
Comment (rosiejeays) [inline src/pages/HomePage/HomePage.jsx:63]: Is this a placeholder until we bring in the events with JS?

**Review by rosiejeays — CHANGES_REQUESTED** [review]

**Comment by rosiejeays** [comment]
Please update the slack link (forgot to add this to my "request changes")

File: src/components/NavBar/NavBar.css
Comment (oriyannadav) [reply src/components/NavBar/NavBar.css:1]: There might be a way but I will have to look at what Andrea has done with her css. So far I haven't been able to do so.

File: src/components/Footer/Footer.jsx (hunk 1 above)
Comment (oriyannadav) [reply src/components/Footer/Footer.jsx]: Have just changed it

File: src/pages/HomePage/HomePage.jsx (hunk 2 above)
Comment (oriyannadav) [reply src/pages/HomePage/HomePage.jsx:63]: Yes

**Review by rosiejeays — APPROVED** [review]
🚢

**Pull request merged**